src/mscorlib/src/System/AggregateException.cs

```diff
@@ -51,7 +51,7 @@ public AggregateException()
         public AggregateException(string message)
             : base(message)
         {
-            m_innerExceptions = new ReadOnlyCollection<Exception>(new Exception[0]);
+            m_innerExceptions = new ReadOnlyCollection<Exception>(Array.Empty<Exception>());
```

Comment:
These constructors of AggregateException are very rarely used (I can't actually remember ever seeing them used), and exist primarily for conformance with the exception design guidelines; if you're constructing an AggregateException, it's because you have some other exceptions to be wrapped, and thus you'd use one of the constructors that takes a collection of exceptions.  So changing these is very unlikely to have any positive impact, and although there's no behavioral/functional risk associated with this change, since it's extremely rare to need an empty array of exceptions, we are probably increasing memory usage slightly if these code paths are ever compiled.  We should revert these.